ci_framework/playbooks/07-run-tests.yml

```diff
@@ -0,0 +1,19 @@
+- name: Run pre_deploy hooks
+  vars:
+    hooks: "{{ pre_tempest | default([]) }}"
+    step: pre_tempest
```

Comment:
let's be generic? `pre_test` (and `post_test`) ?

Also, please add those 2 new hooks in the doc:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/docs/source/usage/01_usage.md#hooks